### ISSUE TITLE
AttributeError: module 'PIL.Image' has no attribute 'Resampling'

### ISSUE THREAD

**Issue by MaxGyver83** [Xpra-org/xpra#3991]
**Describe the bug**

In the output of `xpra attach`, I see this error message:

```
2023-09-06 15:43:42,846 running
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/xpra/client/base/client_base.py", line 1257, in call_handler
    handler(packet)
  File "/usr/lib/python3/dist-packages/xpra/client/mixins/window_manager.py", line 1153, in _process_window_icon
    img = self._window_icon_image(wid, w, h, coding, data)
  File "/usr/lib/python3/dist-packages/xpra/client/mixins/window_manager.py", line 790, in _window_icon_image
    icon_resized = icon.resize((shrunk_width, shrunk_height), Image.Resampling.LANCZOS)
  File "/home/sci6lo/.local/lib/python3.8/site-packages/PIL/Image.py", line 65, in __getattr__
    raise AttributeError(f"module '{__name__}' has no attribute '{name}'")
AttributeError: module 'PIL.Image' has no attribute 'Resampling'
```

**To Reproduce**
1. server command: `xpra start :100 && sleep 5 && export DISPLAY=:100 && xhost +`
2. client command: `xpra attach ssh:workstation:100`
3. specific action to trigger the bug: Open Firefox on the server (maybe not even necessary)

**System Information (please complete the following information):**
 - Server OS: Ubuntu 20.04
 - Client OS: Ubuntu 20.04
 - Xpra Server Version: xpra v5.0.1-r0
 - Xpra Client Version: xpra v5.0.1-r0

**Additional context**
My Pillow version was `9.5.0` on the server and `7.0.0` on the client. On stackoverflow, I found the workaround to install version `9.0.0`:
[python - Module PIL has not attribute &quot;Resampling&quot; - Stack Overflow](https://stackoverflow.com/questions/71738218/module-pil-has-not-attribute-resampling)
I did this both on the client and on the server but I still get the same error.

This error looks similar to #3914.

**Comment by totaam** [comment]
There were many more instances than the one in your PR.

Does the commit above fix things?

**Comment by MaxGyver83** [comment]
> There were many more instances than the one in your PR.
>
> Does the commit above fix things?

Yes, but I didn't want to change things I couldn't test. When I fixed `client/mixins/window_manager.py`, the error message didn't show up anymore. Before, I also noticed that the Firefox window did oscillate (white→grey→white). But now I'm not sure in which version this happened.

**Comment by totaam** [comment]
There was a typo in 35f77d9d9dce78d569f453608aea8b5e712aeb25, fixed in 650e63dc78b98bf71e46f9e647e7d7b9aadb9919
